core/src/main/java/org/frankframework/filesystem/SftpFileSystem.java

```diff
@@ -191,10 +191,19 @@ public void createFolder(String folder) throws FileSystemException {
 		if(folderExists(folder)) {
 			throw new FileSystemException("Create directory for [" + folder + "] has failed. Directory already exists.");
 		}
+
 		try {
-			ftpClient.mkdir(folder);
-		} catch (SftpException e) {
-			throw new FileSystemException(e);
+			String[] folders = folder.split("/");
+			for(int i = 1; i < folders.length; i++) {
+				folders[i] = folders[i - 1] + "/" + folders[i];
+			}
+			for(String f : folders) {
+				if(f.length() != 0 && !folderExists(f)) {
+					ftpClient.mkdir(f);
```

Comment:
Als de directory al bestaat in de ftp server, gooit deze mkdir dan niet iets wat we moeten afvangen?

Reply:
Daar is die folderExist check voor ;)

Reply:
Ah, natuurlijk, ik weet niet hoe ik die over het hoofd gezien heb.